portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,32 @@ function addRandomSaying() {
   const quoteContainer = document.getElementById('quote-container');
   quoteContainer.innerText = quote;
 }
+
+
+var elements = document.getElementsByClassName("column");
+
+// Full-width images
+function one() {
+    for (var i = 0; i < elements.length; i++) {
+    elements[i].style.msFlex = "100%";
+    elements[i].style.flex = "100%";
+  }
+}
+
+// Two images side by side
+function two() {
```

Comment:
Try to balance being succinct and descriptive in your function naming. Calling this something like showTwoImageView() eliminates the need for the comment above and helps with reading your code. This comment also applies for your other functions.